### DISCUSSION TITLE
Comparing the the same file twice: different result ? (version 2.16.12)

### DISCUSSION THREAD

**Discussion by al-dev112233** [WinMerge/winmerge#812] [Q&A] [answered]
As the title says: I compared a backed up (RAR) file twice (using binary comparison): one time it is found as identical the other time: different. I checked the original file and the backed up file itself: the CRC checksum of each RAR file is okay. However: the creation date of the files is different but in a binary compare that should not matter I assume.

**Comment by sdottaka** [comment]
I couldn't reproduce the problem. 

Is my understanding that you compared folders using the Binary Contents method correct?
Or did you view the files in a binary comparison window instead of a folder comparison?

Does the problem reappear only in specific files?

There is a problem in 2.16.12 that the comparison result becomes unstable when comparing folders with the Full Contents method and for files with UTF-16 BOM and no contents. Is this the same as your problem?

https://github.com/WinMerge/winmerge/discussions/768

**Reply by al-dev112233** [reply]
I suspect my antivirus tampering with one of the RAR files during comparison. Sadly I cannot reproduce the result, but will post again when I can.

No, #768 is completely different. I used the binary compare method.

**Comment by al-dev112233** [comment, marked as answer]
It turned out to be a hardware fault (RAM) on the PC I used.
This can be closed then.
